src/archanaNJun20/ExtractItemId.java

```diff
@@ -0,0 +1,29 @@
+package archanaNJun20;
+
+public class ExtractItemId{
+	void printItemId(String input){
+		String output = "";
+		if (input.contains("itemId")) {
+			String temp = input.substring(input.indexOf("itemId"));
+			for (int index = 0; index < temp.length(); index++){
+				if (Character.isDigit(temp.charAt(index))){
+					output += temp.charAt(index);
+					if (!Character.isDigit(temp.charAt(index + 1)))
```

Comment:
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.